### PR TITLE
Torch.hub now based on AutoModels - Updating AutoModels with AutoModelWithLMHead, Sequence Classification and Question Answering

### PR DESCRIPTION
**Pull request by LysandreJik**
Added new AutoModels, as long as the accompanying tests. Updated the TorchHub configuration file to redirect to those AutoModels.

### CODE REVIEW

File: pytorch_transformers/modeling_auto.py [hunk 1]
```diff
@@ -137,39 +137,39 @@ class AutoModel(object):
         when created with the `AutoModel.from_pretrained(pretrained_model_name_or_path)`
         class method.
 
-        The `from_pretrained()` method take care of returning the correct model class instance
+        The `from_pretrained()` method takes care of returning the correct model class instance
         using pattern matching on the `pretrained_model_name_or_path` string.
 
         The base model class to instantiate is selected as the first pattern matching
         in the `pretrained_model_name_or_path` string (in the following order):
+            - contains `roberta`: RobertaModel (RoBERTa model)
```
Comment (thomwolf) [inline pytorch_transformers/modeling_auto.py]: good catch!

**Review by thomwolf — COMMENTED** [review]
Looks good to me! I've updated the docstring for the `from_pretrained` method which now accepts a few additional parameters (`proxies` to add proxies, `force_download` to update the cached files)

**Comment by thomwolf** [comment]
Not sure torch.hub will work directly. We should check with @VictorSanh.

**Comment by LysandreJik** [comment]
Just checked the integration with Pytorch Hub, it works on my end. For example, you can try it out with: 

```python
import torch
torch.hub.load('huggingface/pytorch-transformers:automodels', 'autoModelWithLMHead', 'distilbert-base-uncased')
```

**Comment by thomwolf** [comment]
Ok, I think we are all good with this. Happy to have `torch.hub` integration again.

**Pull request merged**